<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GRegister</class>
 <widget class="QWidget" name="GRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_msg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎注册使用办公辅助软件GGroove!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tel">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>152</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pwd">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>185</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_checkpwd">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>218</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>119</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_uname">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>86</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>191</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*密   码  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>124</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>  Email ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>223</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*重复密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>154</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>  Phone ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>86</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*用户名 ：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reg">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>放弃</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
